doc/ref/configuration/master.rst

```diff
@@ -4597,7 +4597,7 @@ Default: ``3600``
 
 If and only if a master has set ``pillar_cache: True``, the cache TTL controls the amount
 of time, in seconds, before the cache is considered invalid by a master and a fresh
-pillar is recompiled and stored.
+pillar is recompiled and stored. The 0 means always valid.
```

Comment:
Please change this added sentence from `The 0 means always valid.` to `A value of 0 will cause the cache to always be valid.`

Also, please make this change in all three config files that you modified.

Reply:
already fixed